examples/java-on-osg.md

```diff
@@ -0,0 +1,44 @@
+[title]: - "Using Java on the OSG"
+
+[TOC]
+
+## Overview
+
+If your code uses Java via a `.jar` file, it is easy to bring along your 
+own copy of the Java Development Kit (JDK) which allows you to run 
+your `.jar` file anywhere on the OSG. 
+
+## Steps to Use Java on the OSG
+
+1. **Get a copy of Java/JDK.** You can access the the Java Development Kit (JDK) from 
+the [JDK website](https://jdk.java.net/). First select the link to the 
+JDK that's listed as "Ready for Use" and then download the Linux/x64 
+version of the tar.gz file. The downloaded file should end up in your 
```

Comment:
```suggestion
version of the tar.gz file using a Unix command such as `wget` from your `/home` directory. 
For example, 

$ wget https://download.java.net/java/GA/jdk17.0.1/2a2082e5a09d4267845be086888add4f/12/GPL/openjdk-17.0.1_linux-x64_bin.tar.gz

The downloaded file should end up in your 
```

Reply:
I'd put line 20 in a code chunk (GitHub edits won't let me do it in a suggestion).